pkg/cmd/cli/cmd/request_project.go

```diff
@@ -113,23 +116,36 @@ func (o *NewProjectOptions) Run() error {
 		return err
 	}
 
-	if o.ProjectOptions != nil {
-		o.ProjectOptions.ProjectName = project.Name
-		o.ProjectOptions.ProjectOnly = true
-		o.ProjectOptions.SkipAccessValidation = true
+	if !o.SkipConfigWrite {
+		if o.ProjectOptions != nil {
+			o.ProjectOptions.ProjectName = project.Name
+			o.ProjectOptions.ProjectOnly = true
+			o.ProjectOptions.SkipAccessValidation = true
 
-		if err := o.ProjectOptions.RunProject(); err != nil {
-			return err
+			if err := o.ProjectOptions.RunProject(); err != nil {
+				return err
+			}
 		}
-	}
 
-	fmt.Fprintf(o.Out, `
+		fmt.Fprintf(o.Out, `
 You can add applications to this project with the 'new-app' command. For example, try:
 
     %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
 
 to build a new example application in Ruby.
 `, o.Name)
+	} else {
+		if o.ProjectOptions != nil {
+			fmt.Fprintf(o.Out, "Project %q created on server %q.\n", o.ProjectName, o.ProjectOptions.ClientConfig.Host)
+		} else {
+			fmt.Fprintf(o.Out, "Project %q created.\n", o.ProjectName)
+		}
+	}
+
+	fmt.Fprintf(o.Out, `
```

Comment:
Looks like this is unconditional.  Wouldn't it be kind of weird to output this to someone whose config is already updated and they're using this project?